tcms/rpc/api/testexecution.py

```diff
@@ -76,6 +76,24 @@ def remove_comment(execution_id, comment_id=None):
     to_be_deleted.delete()
 
 
+@permissions_required('django_comments.view_comment')
+@rpc_method(name='TestExecution.get_comments')
+def get_comments(execution_id):
+    """
+    .. function:: TestExecution.get_comments(execution_id)
+
+        Get all comments for selected test execution.
+
+        :param execution_id: PK of a TestExecution object
+        :type execution_id: int
+        :raises PermissionDenied: if missing *django_comments.view_comment* permission
+    """
+    execution = TestExecution.objects.get(pk=execution_id)
+    execution_comments = comments.get_comments(execution)
+    serialier = Serializer(execution_comments)
+    return serialier.serialize_queryset()
```

Comment:
Don't use `Serializer`, it's slated for removal. Try list(execution_comments.values()), see Bug.filter() API methods.